web/styles/portico/navbar.css

```diff
@@ -0,0 +1,570 @@
+.top-menu {
+    position: absolute;
+    top: 0;
+    width: 100%;
+    background: hsla(240, 100%, 10%, 0.4);
+    backdrop-filter: blur(10px);
+    font-family: var(--font-ss3);
+    color: hsl(0, 0%, 100%);
+    z-index: 10;
+    overflow: hidden;
+}
+
+.top-menu__container {
+    /*   1280px + 32px for paddings on the edges */
+    max-width: 1312px;
```

Comment:
I think we should try to get into the habit of doing this sort of thing with CSS variables and `calc` calculations, so that it isn't prohibitively difficult to change values like this.

I'm not sure which of the various constants are important to this working, but it'd be great if you can try to extract some of the key constants as variables defined at the top of the file with a comment while this is still fresh. 

(E.g. there's a handful of 3-digit numbers of pixels scattered through this file, and it's a bit hard to guess why any of them have the value they have).